### PR TITLE
Add a metric for checkstyle violations

### PR DESCRIPTION
**Pull request by I-Al-Istannen**
Track checkstyle violations in the Aaaaaaaah-Instance to add some gamification (and get an overview).

### CODE REVIEW

File: bench [hunk 1]
```diff
@@ -68,9 +78,9 @@ def guarded_try_metric(benchmark: str, metric: str, function, unit="seconds", re
 def build_makefile():
     """ Builds VelCom with a makefile. """
     guarded_try_metric(
-        "frontend",
-        "build_time",
-        lambda: execute_program(["make", "-C", str(target_path()), "frontend"])
+       "frontend",
+       "build_time",
+       lambda: execute_program(["make", "-C", str(target_path()), "frontend"])
```
Comment (Garmelon) [inline bench:83]: Why did you change the indentation?

Comment (I-Al-Istannen) [reply bench:83]: By accident. I commented them out for testing

**Review by Garmelon — APPROVED** [review]
